app/modules/integrations/integrations_service.py

```
@@ -213,13 +215,13 @@ async def refresh_sentry_token(self, integration_id: str) -> Dict[str, Any]:
                 response = await client.post(token_url, data=refresh_data)
 
                 if response.status_code != 200:
-                    logging.error(
+                    logger.error(
                         f"Token refresh failed: {response.status_code} - {response.text}"
                     )
                     raise Exception(f"Token refresh failed: {response.status_code}")
```

Ruff (0.14.8)

Abstract `raise` to an inner function

(TRY301)